README.md

```diff
@@ -1,60 +1,48 @@
-# Solidity Documentation Translation Guide
-
-Hello and welcome to the [Solidity documentation](http://docs.soliditylang.org/) translations GitHub organization! Great to have you here. 😊
-
-In this org, we coordinate all community-driven translation efforts. 
```

Comment:
The PR now overwrites the original README. That's not intentional, is it?

Reply:
nope, my bad... let me fix it